packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -638,50 +641,24 @@ contract Deploy is Deployer {
 
     /// @notice Deploy the OptimismPortal
     function deployOptimismPortal() public broadcast returns (address addr_) {
-        if (cfg.useFaultProofs()) {
-            // Could also verify this inside DeployConfig but doing it here is a bit more reliable.
-            require(
-                uint32(cfg.respectedGameType()) == cfg.respectedGameType(),
-                "Deploy: respectedGameType must fit into uint32"
-            );
+        require(!cfg.useFaultProofs(), "Deploy: FaultProofs OptimismPortal is deployed by OPCM");
+        if (cfg.useInterop()) {
+            console.log("Attempting to deploy OptimismPortal with interop, this config is a noop");
+        }
```

Comment:
I know this was already here, but what is the purpose of this `if` block?

Reply:
yeah... IDK, tbh it feels like it should be another require statement. I'll see what happens if I make that change.

Reply:
531d5a4f86ad